docs/reference/migration/migrate_1_6.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[breaking-changes-1.6]]
```

Comment:
wondering, shouldn't this file go in as part of the deprecation PR instead of this one? Also, did we ever keep track of deprecations in migrate* files? this is not really a breaking change for 1.6 and it might create the expectation that for each deprecation we have an entry in the migrate document, which I think we haven't.